tests/test_data/tabs_and_spaces.robot

```diff
@@ -0,0 +1,21 @@
+*** Settings ***
+Library    OperatingSystem
+Documentation	Suite documentation.
+
+
+*** Variables ***
+@{RANDOM_LIST}    first_element	second_element
+
+
+*** Test Cases ***
+Test Which Includes Tabs And Spaces
+    [Documentation]    Test documentation.
+    Log Two Elements To Console    ${RANDOM_LIST}[0]    ${RANDOM_LIST}[1]
+
+
+*** Keywords ***
+Log Two Elements To Console
+    [Documentation]	Keyword documentation.
+    [Arguments]    ${first_element}    ${second_element}
+    Log To Console    ${first_element}	${second_element}
+    Log To Console	    ${second_element}    	${first_element}
```

Comment:
This file will also log issue with no blank line at the end of file. Please correct 😉

Reply:
My bad, IDE configuration issue. :)